tests/suite/ndjson/empty-objects.yaml

```
@@ -0,0 +1,10 @@
+zql: '*'
+
+input: |
+  {}
+  {"ja3s":{}}
+
+output-format: ndjson
+output: |
+    {}
+    {"ja3s":{}}
```

Nit: Indent this block scalar by two spaces as we do elsewhere.
```suggestion
output: |
  {}
  {"ja3s":{}}
```